ansible/group_vars/all

```diff
@@ -125,3 +125,4 @@ mn_evo_services_path: '{{ dashd_home }}/{{ mn_evo_services_compose_project_name
 elastic_username: elastic
 elastic_password: 
 kibana_encryptionkey: 
+elastic_max_heap_size: 
```

Comment:
What is the minimal recommended heap size? I would keep something default here.

Reply:
Default is 512MB, recommended is 50% of system memory. We already have to define the above variables before deploy is possible, so maybe just add a comment here about requirements and let the user define it depending on the node instance they select?

Reply:
We have default instance for logs so we know default memory size, right? Why not just put it here. If you change the instance you need fo change heap size as well.